sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -131,11 +132,16 @@ public KeyVaultKeyStore() {
                     .map(Boolean::parseBoolean)
                     .orElse(false);
         jreCertificates = JreCertificates.getInstance();
+        LOGGER.log(FINE, String.format("Loaded jre certificates: %s.", jreCertificates.getAliases()));
         wellKnowCertificates = SpecificPathCertificates.getSpecificPathCertificates(wellKnowPath);
+        LOGGER.log(FINE, String.format("Loaded well know certificates: %s.", wellKnowCertificates.getAliases()));
```

Comment:
wellKnown

Reply:
Shouldn't this be well known? Well know is not correct, right?

Reply:
New PR created: https://github.com/Azure/azure-sdk-for-java/pull/27426